lib/linguist/heuristics.yml

```diff
@@ -88,6 +88,16 @@ disambiguations:
     pattern: '(<^\s*; |End Function)'
   - language: BitBake
     pattern: '^\s*(# |include|require)\b'
+- extensions: ['.bf']
+  rules:
+  - language: Beef
+    pattern: '^using\s+(System|Beefy)(\.(.*))?;\s*$'
```

Comment:
Wouldn't just `^using\s+(System|Beefy)` work fine? Also should there be a `\s*` before `using`?